mmcls/core/evaluation/eval_hooks.py

```diff
@@ -12,21 +12,30 @@ class EvalHook(Hook):
         interval (int): Evaluation interval (by epochs). Default: 1.
     """
 
-    def __init__(self, dataloader, interval=1, **eval_kwargs):
+    def __init__(self, dataloader, interval=1, by_epoch=False, **eval_kwargs):
```

Comment:
Better to set the default value to True to keep consistent with the old behavior?